### PR TITLE
New Class for media type naming and updated response naming

### PR DESCRIPTION
**Pull request by colbyhemond**
add method to condense content type, updated return structure component naming

This fix currently only supports json and xml content types, and using the full content type name for all others. Should I go through all other content types to make sure those can be handled too, so that the 30 character naming limit doesn't cause confusion by truncating any words?

### CODE REVIEW

File: src/v2/zcl_oapi_generator_v2.clas.abap [hunk 1]
```diff
@@ -328,4 +342,15 @@ CLASS zcl_oapi_generator_v2 IMPLEMENTATION.
       ENDLOOP.
     ENDLOOP.
   ENDMETHOD.
+
+
+  METHOD shorten_content_type_name.
+    rv_short_name =
+      SWITCH #( iv_content_type
```
Comment (larshp) [inline src/v2/zcl_oapi_generator_v2.clas.abap:349]: this SWITCH must be downported to 702 syntax

Comment (colbyhemond) [reply src/v2/zcl_oapi_generator_v2.clas.abap]: I have removed this method. It was replaced with a class that handles the majority of the media types instead.

**Comment by larshp** [comment]
I'll have a look sometime, just didnt get to it yet...

**Comment by colbyhemond** [comment]
https://github.com/abap-openapi/abap-openapi/issues/52

**Comment by colbyhemond** [comment]
> I'll have a look sometime, just didnt get to it yet...

No rush, I was just working on keeping it connected with the issue and up to date 👍

**Pull request merged**